updateTablesToNewUser.js

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable guard-for-in */
+/* eslint-disable no-restricted-syntax */
+const AWS = require('aws-sdk');
+
+// Configure the AWS SDK with your credentials
+AWS.config.update({
+  accessKeyId: 'your_access_key',
+  secretAccessKey: 'your_secret_access_key',
+  region: 'your_region',
+});
+
+// Create an instance of the AWS Cognito Identity provider client
+const cognitoIdentityServiceProvider = new AWS.CognitoIdentityServiceProvider();
+
+// The ID of the Cognito User Pool you want to retrieve users from
+const userPoolId = 'us-east-1_lfq00ab2u';
```

Comment:
Is this an actual userPoolId ? If so, I suggest removing it.

Reply:
Oops, I removed this out of the commit entirely.